VK/VK/Extension/ViewController+extension.swift

```diff
@@ -0,0 +1,29 @@
+// ViewController+extension.swift
```

Comment:
```suggestion
// LoginViewController+extension.swift
```
Не то название файла